src/business-logic/services/tv2-inews-ingest-service.ts

```diff
@@ -0,0 +1,12 @@
+import { IngestService } from './interfaces/ingest-service'
+import {HttpRequestParameters, HttpService} from './interfaces/http-service'
+
+export class Tv2InewsIngestService implements IngestService {
+  constructor(private readonly httpService: HttpService) {}
+  public async reloadIngestData(rundownName: string): Promise<void> {
+    const domain = process.env.INEWS_DOMAIN ?? 'localhost:3007'
+    const url = `http://${domain}/reloadData/${rundownName}`
```

Comment:
This should technically be on the iNews-gateway PR, but we are not following REST standards here with this URL.
We can't follow the standard completely since we are not using an id for the Rundown, but the URL should either be `/reloadData/rundowns/{rundownName}` or `/rundowns/{rundownName}/reloadData` dependeing on how you view the `reloadData` operation.

Reply:
I have changed it to use `/reloadData/rundowns/{rundownName}`.